CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2015-08-18 Release 1.2.0
+
+### Backwards-incompatible changes
+
+None
+
+### Summary
+
+Enhanced support for Amazon Linux
+Ability to set HOME environment variable when installing npm
+Misc lint/spec fixes
```

Comment:
This should probably be a single sentence or three bullet points, it'll render weirdly like this.